assignment5.html

```diff
@@ -38,7 +38,7 @@ <h1>Sage Bistro Static Google Map</h1>
                     </p>
                 </div>
                 <div class="ui-block-b">
```

Comment:
Nice grid + block!

This looks like you used my API key. I would then expect some screenshots either here in your repo or in your BB submission showing that you created your own credentials without billing attached.